Frontend/.env.example

```
@@ -0,0 +1,41 @@
+VITE_OPENROUTER_API_KEY_3=
+VITE_OPENROUTER_API_KEY_4=
+# Comma-separated OpenRouter models to use
+VITE_AI_OPENROUTER_MODELS=meta-llama/llama-3.2-3b-instruct:free,microsoft/phi-3-mini-128k-instruct:free,mistralai/mistral-7b-instruct:free,google/gemma-2-9b-it:free
+
+# --- AI Provider: Groq API Keys (optional) ---
+# Up to 3 keys for load balancing. Get keys at https://console.groq.com/keys
+VITE_GROQ_API_KEY_1=
+VITE_GROQ_API_KEY_2=
+VITE_GROQ_API_KEY_3=
+# Comma-separated Groq models to use
+VITE_AI_GROQ_MODELS=llama-3.1-8b-instant,mixtral-8x7b-32768,gemma2-9b-it
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_AI_GROQ_MODELS key should go before the VITE_GROQ_API_KEY_1 key

(UnorderedKey)